README.md

```diff
@@ -935,6 +935,18 @@ expect(message).to eq({
   topic: 'my-topic',
   key: 'my-id'
 })
+
+## Configuring Deimos to test settings
+
+# You can reset Deimos to default test config by calling this method.
+configure_deimos
+
+# The same method can be used to override settings while still using
+# deafults for other fields.
+configure_deimos(consumers: { reraise_errors: false },
+                 producers: { topic_prefix: nil },
+                 db_producer: { compact_topics: %w(my-topic my-topic2) },
+                 logger: Rails.logger)
```

Comment:
Nice, I like this a lot! I think there might be some other changes to the README that are necessary - e.g. we should be calling out things like the test producer and how to interact with it rather than assuming that when you include TestHelpers you get it for free.

Reply:
@dorner I tried my best to add more info but there might be some that I missed and/or added incorrect information. Please let me know!